### PR TITLE
Branch4

### PR DESCRIPTION
**Pull request by grantye**
Navbar changed with dynamic working. However, I couldn't log in because for some reason I have been set as "user" by default, neither "teacher" nor "professional". Even if I destroy all the data and re-register, I am still a "user". Probably needs to fix it ASAP.

### CODE REVIEW

**Comment by frank657** [comment]
> Navbar changed with dynamic working. However, I couldn't log in because for some reason I have been set as "user" by default, neither "teacher" nor "professional". Even if I destroy all the data and re-register, I am still a "user". Probably needs to fix it ASAP.

We had decided to not use STI and instead work with profile models. It means when you create a user, you need to either create a professional or a teacher. The best is to run the seed and change one of the users(teacher or professional) into a user with the attributes you want. Then it should work.

File: db/schema.rb [hunk 1]
```diff
@@ -40,23 +40,25 @@
...
     t.string "company"
     t.string "title"
     t.string "description"
     t.string "city"
     t.string "linkedin_url"
+    t.bigint "user_id"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index ["user_id"], name: "index_professionals_on_user_id"
   end
```
Comment (frank657) [inline db/schema.rb]: Why did you change the schema? @grantye

**Review by frank657 — COMMENTED** [review]
Please reply asap, we need to finish the landing page.

File: db/schema.rb (hunk 1 above)
Comment (grantye) [reply db/schema.rb]: I didn't change any schema. Were they from the old versions?
